frontend/src/services/user.service.ts

```diff
@@ -10,19 +10,28 @@ export const getDatabaseUser = async (id: number): Promise<any> =>
   await authFetch(`${NEST_URI}/user/${id}`)
 
 /**
- * Do we get the spotify user from spotify authentication?
+ * TODO: Do we get the spotify user from spotify authentication?
  * @returns
  */
 export const getSpotifyUser = async (): Promise<any> =>
   await authFetch(`${NEST_URI}/spotify`)
 
 /**
- * Don't we already get all this information from authenticating in spotify.
- * Do we really need to make an request?
+ * TODO: Don't we already get all this information from authenticating in spotify.
```

Comment:
If you can check that out real quick, by console.logging this response and then authentication response. ALso check if this even gets used or Mabey its useless